python/perf-kernels/flash-attention.py

```diff
@@ -563,10 +688,13 @@ def forward(ctx, q, k, v, causal, bias, sm_scale, split_kernel=False):
         o = torch.empty_like(q, dtype=v.dtype)
         M = torch.empty((q.shape[0] * q.shape[1], q.shape[2]), device=q.device, dtype=torch.float32)
 
+        philox_seed = 0x1BF52
```

Comment:
I assume these are random constants chosen for reproducibility? Comments for these magic numbers would be helpful.